Daytime-Nighttime-Interactions/daytime-and-nighttime-communication.rst

```diff
@@ -57,6 +57,7 @@ Afternoon checkouts are important procedures to ensure that the telescopes and i
 The full procedures for the different afternoon or recommended daily checkouts are described on these pages:
 
 * AuxTel daily checkout
+    * `AuxTel daily checkout procedure notebook <https://github.com/lsst-ts/ts_notebooks/blob/develop/procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb>`__
 * LATISS calibrations
     * `LATISS master calibrations generation procedure <https://obs-ops.lsst.io/Operational-Procedures/Auxiliary-Telescope/LATISS-Master-Calibrations-Procedure/latiss-master-calibrations-procedure.html>`__
```

Comment:
I know this is not part of this PR but you can probably use an internal link here, e.g.;

```
:ref:`AuxTel daily checkout procedure notebook <Latiss-Master-Calibrations-Procedure>`.
```